ci-operator/step-registry/vsphere/remote-worker/vsphere-remote-worker-commands.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+infrastructure_id="$(oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster)"
+template="$(oc get machineset -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.template}' -n openshift-machine-api)"
+datacenter="$(oc get machineset -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.workspace.datacenter}' -n openshift-machine-api)"
+datastore="$(oc get machineset -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.workspace.datastore}' -n openshift-machine-api)"
+folder="$(oc get machineset -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.workspace.folder}' -n openshift-machine-api)"
+resourcePool="$(oc get machineset -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.workspace.resourcePool}' -n openshift-machine-api)"
+server="$(oc get machineset -o jsonpath='{.items[0].spec.template.spec.providerSpec.value.workspace.server}' -n openshift-machine-api)"
+
+cat >> "${SHARED_DIR}/remote-worker-machineset.yaml" << EOF
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: ${infrastructure_id}
+  name: remote-worker
+  namespace: openshift-machine-api
+spec:
+  replicas: 2
```

Comment:
suggest to introduce new ENV (e.g "REMOTE_WORKER_NUMBER" ) instead of hardcode here

Reply:
Fixed